<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1075</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1,0">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../icons/cil-magnifying-glass.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_search">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Search names...</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
     <item>
      <layout class="QHBoxLayout" name="workspace_layout">
       <item>
        <widget class="QWidget" name="parts_widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="parts_layout"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="assembly_widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="assembly_layout"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_40">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_23">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="workspace_side_panel_2" native="true">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="workspace_side_panel">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QPushButton" name="pushButton_view_parts">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>View Parts</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_view_assemblies">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>View Assemblies</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QScrollArea" name="scrollArea_5">
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_6">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>325</width>
                 <height>320</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_21">
                <item>
                 <layout class="QVBoxLayout" name="filter_layout"/>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_due_dates">
              <property name="title">
               <string>Timeline</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_35" stretch="1,0">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_calendar_select_range"/>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_16">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="title">
                  <string>Quick Select Timeline</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_39">
                  <item>
                   <widget class="QLabel" name="label_13">
                    <property name="font">
                     <font>
                      <italic>true</italic>
                      <bold>false</bold>
                     </font>
                    </property>
                    <property name="text">
                     <string>* With respect to the current date</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_38">
                    <property name="spacing">
                     <number>3</number>
                    </property>
                    <item>
                     <layout class="QHBoxLayout" name="horizontalLayout_7">
                      <item>
                       <widget class="QLabel" name="label_15">
                        <property name="minimumSize">
                         <size>
                          <width>40</width>
                          <height>0</height>
                         </size>
                        </property>
                        <property name="maximumSize">
                         <size>
                          <width>40</width>
                          <height>16777215</height>
                         </size>
                        </property>
                        <property name="text">
                         <string>Due: </string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pushButton_next_2_days">
                        <property name="text">
                         <string>Next 2 Days</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pushButton_next_4_days">
                        <property name="text">
                         <string>Next 4 Days</string>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QPushButton" name="pushButton_this_week">
                        <property name="text">
                         <string>This Week</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_view_assemblies</sender>
   <signal>toggled(bool)</signal>
   <receiver>assembly_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>973</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_view_parts</sender>
   <signal>toggled(bool)</signal>
   <receiver>parts_widget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>807</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>390</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
